wampy/transports/websocket/frames.py

```diff
@@ -248,13 +259,17 @@ def __init__(self, bytes):
             )
 
         self.opcode = bytes[0] & 0b1111
-        try:
-            # decode required before loading JSON for python 2 only
-            self.payload = json.loads(self.body.decode('utf-8'))
-        except Exception:
-            raise WebsocktProtocolError(
-                'Failed to load JSON object from: "%s"', self.body
-            )
+
+        if self.opcode != 9:
```

Comment:
a brief comment here too for those who aren't familiar with the websocket protocol. thanks